### ISSUE TITLE
SyntaxError: Named export 'fromBuffer' not found.

### ISSUE THREAD

**Issue by boringContributor** [dan-online/bunnycdn-stream#86]
Hey there,

first off all BIG thanks for this project. I made a video streaming backend and created the requests manually, but switched now to your client as it makes my life much easier.

However, I'm having a problem:

I'm using NextJS (13.1.6) API routes and get the following error:

![image](https://user-images.githubusercontent.com/23439129/219901585-1d1a8802-6961-421d-899c-57e36a19e20e.png)

My code looks like this:
```
export default withUserGuard(async function handler(req, res) {
    if (req.method !== 'GET') {
        res.setHeader("Allow", ["GET"]);
        return res.status(405).json({ error: `Method ${req.method} Not Allowed` });
    }
    try {
        const mostRecent = await getMostRecentVideos();
        return res.status(200).json({ mostRecent });
    } catch (e) {
        return res.status(500).end();
    }
});

const getMostRecentVideos = async () => {
    const videos = await stream.listVideos({
        page: 1,
        itemsPerPage: 10,
        orderBy: 'date'
    })

    return videos.items.map((video) => ({
        viewTime: prettyPrintSeconds(video.length),
        collection: collectionIdToGenre.get(video.collectionId),
        ...video,
    }));
};
```

Do you have an idea how to solve that or is there a way to allow CommonJS modules properly?
Thanks and all the best!

**Comment by boringContributor** [comment]
I checked your code and tried the following:

![image](https://user-images.githubusercontent.com/23439129/219901801-166412bb-89ed-4809-bfa1-3e5a5015973c.png)

so there is an issue somewhere. If you have an idea -> great, if not I'd think about it and contribute when I found the issue :)

**Comment by Abdulazeez-98** [comment]
> I checked your code and tried the following:
> 
> ![image](https://user-images.githubusercontent.com/23439129/219901801-166412bb-89ed-4809-bfa1-3e5a5015973c.png)
> 
> so there is an issue somewhere. If you have an idea -> great, if not I'd think about it and contribute when I found the issue :)

try fileType.fromBuffer() this would theoretically solve the issue. I will create a pull request regarding this.

**Comment by boringContributor** [comment]
@Abdulazeez-98 yes the screenshot above is of the npm package. I modified it and it worked so yeah not sure why it bundles it like that but it does not work. Thanks for the PR!

**Comment by Abdulazeez-98** [comment]
@boringContributor 
You're welcome 👍
The library "file-type" doesn't export individual objects because it only supports CommonJS (or at least that's my understanding). So we need to import the whole thing and then call the function fromBuffer().
This is a limitation in the "file-type" library for not supporting ESM imports. There's a new version of this library but it only supports ESM and not CommonJS. Most NPM libraries support both module systems.

**Comment by dan-online** [comment]
Ah yes the usual discrepancies between cjs and esm, personally I promote the use of esm for everything but I understand why you would need a cjs bundle. Tad annoying that libraries like this don't support both 😕 

Thanks @Abdulazeez-98 for the PR, I'll test it and get that merged if everything looks good

**Comment by dan-online** [comment]
Published in ``bunnycdn-stream@2.1.3``
